docker-images/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.7-slim-buster
+
+RUN apt-get -y update && apt-get -y upgrade
+RUN apt-get -y install git clang build-essential
+
+COPY ./ SyMPC/
+
+WORKDIR /SyMPC
+
+RUN pip3 install setuptools_scm
```

Comment:
Q: Is this required?

Reply:
Yeah it's required, it was a fix for some bug that was coming.